lib/modules/us_core_guidance/clinicalnotes_sequence.rb

```diff
@@ -198,16 +189,17 @@ def test_clinical_notes_diagnostic_report(category_code)
         test_clinical_notes_diagnostic_report('http://loinc.org|LP29684-5')
       end
 
-      test 'DiagnosticReport and DocumentReference reference the same attachment' do
+      test :have_matched_attachments do
         metadata do
-          id '10'
+          id '09'
+          name 'DiagnosticReport and DocumentReference reference the same attachment'
           link 'https://www.hl7.org/fhir/us/core/clinical-notes-guidance.html'
           description %(
           )
           versions :r4
         end
 
-        skip 'No Clinical Notes appear to be available for this patient. Please use patients with more information.' unless @clinical_notes_found
+        skip 'Not all required Clinical Notes appear to be available for this patient. Please use patients with more information.' if @skip_document_reference || @skip_diagnostic_report
```

Comment:
I don't see why this test has to be skipped if a single category of note/report is missing.